neo/io/neuralynxio.py

```diff
@@ -52,24 +56,39 @@ def __init__(
         cache_path : str, optional
             Folder path to use for cache files.
             Default: 'same_as_resource'
-        exclude_filename: str or list
+        exclude_filename: None,
+            Deprecated and will be removed. Please use `exclude_filenames` instead
+        include_filenames: str or list
+            Filename or list of filenames to be included. This can be absolute path or path relative to dirname.
+        exclude_filenames: str or list
             Filename or list of filenames to be excluded. Expects base filenames without
             directory path.
         keep_original_times : bool
             Preserve original time stamps as in data files. By default datasets are
             shifted to begin at t_start = 0*pq.second.
             Default: False
         """
+
+        if filename:
```

Comment:
It's actually a bit better to say 
```
if filename is not None
```
 Since we are explicitly making it None and so the user would have to actively change this.